docs/admin/administration.md

```diff
@@ -947,3 +947,8 @@ required that all engines in the same system run the same DAOS version.
 
 !!! warning
     Rolling upgrade is not supporting at this time.
+
+DAOS v2.2 client connections to pools which were created before DAOS v2.4
```

Comment:
Not sure about "which were created before DAOS v2.4".
"which were created **by** DAOS v2.4"?